app/views/pruebas/edit.html.erb

```diff
@@ -0,0 +1,8 @@
+<div class='columns'>
+    <div class='column'></div>
+    <div class='column'>
+        <h1 class="title is-1">Editar</h1>
```

Comment:
I think `is-1` is for column size. Otherwise `is-size-1` can help you to set a text size.